lib/src/date-input/DateInput.tsx

```diff
@@ -11,30 +11,32 @@ import useTheme from "../useTheme";
 import DxcTextInput from "../text-input/TextInput";
 import DateInputPropsType, { RefType } from "./types";
 
+const getValueForPicker = (value, format) => moment(value, format.toUpperCase(), true).format();
+
 const DxcDateInput = React.forwardRef<RefType, DateInputPropsType>(
   (
     {
       label,
-      name = "",
+      name,
+      defaultValue = "",
       value,
       format = "dd-MM-yyyy",
       helperText,
       placeholder = false,
-      clearable = false,
-      disabled = false,
-      optional = false,
+      clearable,
+      disabled,
+      optional,
       onChange,
       onBlur,
       error,
-      autocomplete = "off",
+      autocomplete,
       margin,
-      size = "medium",
-      tabIndex = 0,
+      size,
+      tabIndex,
```

Comment:
Why have you removed these default values?

Reply:
The Date Input uses internally the Text Input, so the default values for each prop are already there, we were rewriting the same code.

Reply:
I only did this to the props that are passed directly to the text input and already have a default value (and match the value).